augur/tasks/github/detect_move/core.py

```diff
@@ -21,15 +31,27 @@ def update_repo_with_dict(repo,new_dict,logger):
             logger: logging object
             db: db object
     """
-    
-    to_insert = repo.__dict__
+    to_insert = dict(repo.__dict__)
     del to_insert['_sa_instance_state']
+
+    old_url = to_insert["repo_git"]
+    repo_id = to_insert["repo_id"]
+
+    with DatabaseSession(logger) as session:
+        previous_alias = RepoAlias(repo_id=repo_id, git_url=old_url)
+        try:
+            result = session.add(previous_alias)
+            session.commit()
+        except IntegrityError as e: #Unique violation
+            session.rollback()    
+
     to_insert.update(new_dict)
 
     result = bulk_insert_dicts(logger, to_insert, Repo, ['repo_id'])
 
     url = to_insert['repo_git']
-    logger.info(f"Updated repo for {url}\n")
+    logger.info(f"Updated repo {old_url} to {url} and set alias\n")
```

Comment:
In the event that an `IntegrityError` occurs in the above try/catch, this log statement becomes untrue.

Either the conflict must be resolved above before continuing, or a separate log statement must be issued when setting an alias fails.